.gitlab-ci.yml

```diff
@@ -1,76 +1,33 @@
-image: ubuntu:bionic
+image: coqci/base:V2018-05-03
 
 stages:
-  - opam-boot
   - build
   - test
 
 # some default values
 variables:
-  # Format: $IMAGE-V$DATE-$HOUR-$MINUTE
-  CACHEKEY: bionic-V2018-04-29-00-50
-  DEBIAN_FRONTEND: "noninteractive"
-  NJOBS: "2"
+  # Format: $IMAGE-V$DATE
+  CACHEKEY: bionic_coq-V2018-05-03
+  # This shouldn't be needed here, but for some reason they are not
+  # properly picked from the docker image.
   COMPILER: "4.02.3"
```

Comment:
Why do we still set this here but not in circle?

Reply:
I thought about this, but is it possible to set that in circle altogether? The `save-cache` entry is gone, so I am not sure we can do this.

I prefer to have this info here for now as to document wha'ts going on in case the cache resurrects in the future.

On the other hand, we need to discuss the future or CircleCI.